doc/userguide/src/Appendices/CommandLineOptions.rst

```diff
@@ -48,6 +49,7 @@ Command line options for test execution
                            *Statistics by Suite* table in outputs.
   --tagstatinclude <tag>  `Includes only these tags`_ in the *Statistics by Tag* table.
   --tagstatexclude <tag>  `Excludes these tags`_ from the *Statistics by Tag* table.
+  --tagstatskip <tag>     `Skips these tags`_ from the *Statistics by Tag* table.
```

Comment:
No worries about the delay. Paid work obviously has higher priority than open source contributions (unless you are paid for the latter).

I know implementing skipping correctly is not easy. The biggest problem I see with it is designing it so that it works consistently with existing `--exclude/--include` and `--critical/--noncritical` functionality. I hope you understand I'm not going to spend time with it unless someone pays me. Currently this isn't an important feature for any of my paying clients.